app/models/concerns/health_charts.rb

```diff
@@ -155,7 +164,7 @@ def self.health_housing_outcome(answer)
     end
 
     def self.health_housing_score(answer)
-       health_housing_outcomes[answer].try(:[], :score) || 0
+       health_housing_outcomes[answer].try(:[], :score) # || 0
```

Comment:
Remove the comment -- it makes me nervous since it will change the behavior of all the loops above!